src/helpers/globalCss.ts

```diff
@@ -50,7 +50,493 @@ export const core = css`
+  --mdhui-blue-vivid-90-base: rgb(0, 26, 72);
+  --mdhui-blue-vivid-95-base: rgb(0, 24, 67);
+  --mdhui-blue-vivid-96-base: rgb(0, 23, 63);
+  --mdhui-blue-vivid-97-base: rgb(0, 20, 56);
+  --mdhui-blue-vivid-98-base: rgb(0, 18, 50);
+  --mdhui-blue-vivid-99-base: rgb(0, 16, 45);
+
+  --mdhui-indigo-1-base: rgb(251, 252, 255);
+  --mdhui-indigo-2-base: rgb(247, 248, 255);
+  --mdhui-indigo-3-base: rgb(244, 245, 255);
+  --mdhui-indigo-4-base: rgb(241, 241, 255);
+  --mdhui-indigo-5-base: rgb(237, 237, 255);
+  --mdhui-indigo-10-base: rgb(222, 221, 255);
+  --mdhui-indigo-20-base: rgb(200, 196, 253);
+  --mdhui-indigo-30-base: rgb(172, 163, 250);
+  --mdhui-indigo-40-base: rgb(143, 130, 246);
+  --mdhui-indigo-50-base: rgb(112, 94, 243);
+  --mdhui-indigo-55-base: rgb(102, 83, 240);
+  --mdhui-indigo-60-base: rgb(83, 61, 224);
+  --mdhui-indigo-70-base: rgb(54, 28, 201);
+  --mdhui-indigo-80-base: rgb(36, 16, 142);
+  --mdhui-indigo-90-base: rgb(22, 8, 98);
+  --mdhui-indigo-95-base: rgb(21, 7, 92);
+  --mdhui-indigo-96-base: rgb(19, 7, 86);
+  --mdhui-indigo-97-base: rgb(17, 6, 78);
+  --mdhui-indigo-98-base: rgb(16, 6, 71);
+  --mdhui-indigo-99-base: rgb(14, 5, 63);
+
+  --mdhui-cyan-1-base: rgb(247, 253, 254);
+  --mdhui-cyan-2-base: rgb(237, 251, 253);
+  --mdhui-cyan-3-base: rgb(228, 249, 252);
+  --mdhui-cyan-4-base: rgb(219, 247, 251);
+  --mdhui-cyan-5-base: rgb(209, 245, 250);
+  --mdhui-cyan-10-base: rgb(168, 235, 246);
+  --mdhui-cyan-20-base: rgb(93, 218, 239);
+  --mdhui-cyan-30-base: rgb(63, 188, 212);
+  --mdhui-cyan-40-base: rgb(34, 159, 185);
+  --mdhui-cyan-50-base: rgb(4, 128, 158);
+  --mdhui-cyan-55-base: rgb(10, 118, 148);
+  --mdhui-cyan-60-base: rgb(10, 99, 124);
+  --mdhui-cyan-70-base: rgb(9, 74, 94);
+  --mdhui-cyan-80-base: rgb(8, 50, 65);
+  --mdhui-cyan-90-base: rgb(8, 31, 42);
+  --mdhui-cyan-95-base: rgb(8, 29, 39);
+  --mdhui-cyan-96-base: rgb(7, 27, 36);
+  --mdhui-cyan-97-base: rgb(6, 24, 32);
+  --mdhui-cyan-98-base: rgb(6, 22, 28);
+  --mdhui-cyan-99-base: rgb(5, 18, 24);
+
+  --mdhui-green-1-base: rgb(248, 253, 251);
+  --mdhui-green-2-base: rgb(241, 251, 246);
+  --mdhui-green-3-base: rgb(234, 249, 242);
+  --mdhui-green-4-base: rgb(230, 247, 240);
+  --mdhui-green-5-base: rgb(219, 244, 233);
+  --mdhui-green-10-base: rgb(189, 234, 215);
+  --mdhui-green-20-base: rgb(138, 217, 184);
+  --mdhui-green-30-base: rgb(69, 194, 143);
+  --mdhui-green-40-base: rgb(35, 164, 130);
+  --mdhui-green-50-base: rgb(1, 132, 116);
+  --mdhui-green-55-base: rgb(0, 123, 108);
+  --mdhui-green-60-base: rgb(0, 103, 91);
+  --mdhui-green-70-base: rgb(0, 78, 69);
+  --mdhui-green-80-base: rgb(0, 52, 47);
+  --mdhui-green-90-base: rgb(0, 33, 30);
+  --mdhui-green-95-base: rgb(0, 31, 29);
+  --mdhui-green-96-base: rgb(0, 29, 27);
+  --mdhui-green-97-base: rgb(0, 26, 23);
+  --mdhui-green-98-base: rgb(0, 23, 21);
+  --mdhui-green-99-base: rgb(0, 20, 19);
+}
+
+:root {
+  /* Light mode questions */
```

Comment:
Is this comment accurate?  I'm not sure what it means.  Should there be a "dark mode" counterpart?